ClientLib/src/main/java/com/o3dr/android/client/apis/MissionApi.java

```diff
@@ -128,4 +128,16 @@ private Action buildComplexMissionItem(Bundle itemBundle) {
         else
             return null;
     }
+
+    public void pauseMission(AbstractCommandListener listener) {
```

Comment:
@chaviw can you add javadocs to describe each newly added method.
Also include the `@since` javadocs tag to specify which version of the api they were added in?